include/libdnf/rpm/package_query.hpp

```diff
@@ -587,6 +587,16 @@ class PackageQuery : public PackageSet {
     // TODO(dmach): consider removing the installed packages during the filtering
     void filter_priority();
 
+    /// Keep in query only packages that are installed but not available in any enabled
```

Comment:
May I ask you to add a note that filter ignores regular excludes?

Reply:
oops. Sorry, I missed this comment. Should be fixed now.